### DISCUSSION TITLE
useForm isn't function when doing SSR

### DISCUSSION THREAD

**Discussion by gobielJonathan** [react-hook-form/react-hook-form#9759] [Q&A]
Hello guys, I got error `useForm isn't function` when doing SSR, this evidence of the error 
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/46088334/211954246-91b8bd76-aee9-4c07-9dde-c32131484943.png">

what should i do to prevent this? is there anything steps that i skipped or missing?

**Comment by leapful** [comment]
@gobielJonathan 

Please following the template to ask question here : https://github.com/react-hook-form/react-hook-form/issues/2848. Without information or codesandbox url to re-produce the problem, there is no way anyone can help you.
